README.md

```diff
@@ -1 +1,194 @@
-# react-app
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+**[Math-magicians(react-app)]** is an app where people(math entusiests) can use a functioning calculator, and even add comments/messages
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **React**
+- **HTML**
+- **Webpack**
+- **Github Setup**
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Listed below are the key features of my webpage.
+
+- **Setup**
+- **Calculator UI**
+- **Display QUote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://danielmatama.github.io/react-app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Terminal
+- Git
+- IDE
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/danielmatama/react-app.git
+
+  cd matama-awesome-es6
+```
+-
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+-
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  Open this project in your live server
+```
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+
+👤 **Daniel Matama**
+
+- GitHub: [DanielMatama](https://github.com/danielmatama)
+- LinkedIn: [DanielMatama](https://www.linkedin.com/in/daniel-matama-9190ba254/)
+
```

Comment:
- [OPTIONAL] It would be a good idea if you add the contact info of @AbdimulhinYussuf3675 as well to the README file as he worked on the project 😁 😁